.github/workflows/query-engine-driver-adapters.yml

```diff
@@ -0,0 +1,97 @@
+name: Driver Adapters
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
```

Comment:
Should we ignore changes in `schema-engine`, `prisma-fmt` etc here?

Reply:
I think the same rules of the query engine tests  should apply in here.  Currently those are not ignoring other paths, because changes in the schema engine _might trigger_ (IIRC from a conversation with @tomhoule) new behaviors in the query-engine.

Having said that, I reckon we want to skip builds as much as possible and address both ignored paths in `query-engine.yml` and `query-ending-driver-adapters.yml ` and we should visit this optimization as part of the technical-debt key result that we have.